src/hypothesis/strategies.py

```diff
@@ -885,10 +875,7 @@ def decimals(min_value=None, max_value=None,
         # Fixed-point decimals are basically integers with a scale factor
 
         def try_quantize(d):
-            try:
-                return d.quantize(factor)
-            except InvalidOperation:  # pragma: no cover
-                return None
+            return d.quantize(factor)
```

Comment:
Sorry, those were some debugging changes from earlier work that accidentally ended up in this branch. I've backed them out.